tests/test_topics.py

```diff
@@ -0,0 +1,180 @@
+import networkx as nx
+import numpy as np
+import pytest
+from gensim.models import KeyedVectors
+from gensim.scripts.glove2word2vec import glove2word2vec
+from nltk.corpus import wordnet as wn
+
+from extra_model._topics import (
+    aggregate,
+    filter_aggregates,
+    get_nodevec,
+    has_connection,
+    iterate,
+    path_to_graph,
+    traverse_tree,
+)
+from extra_model._vectorizer import Vectorizer
+
+
+@pytest.fixture()
+def minivec():
```

Comment:
I've added a way to clean-up temporary files here, so makes sense to change here as well - https://github.com/wayfair-incubator/extra-model/blob/main/tests/test_vectorizer.py

I'll consider moving it to different scope since we have it in 3 places already, so makes sense to refactor.

Reply:
Followed your cleanup example for now
Feel free to refactor, there are more instances of this